.changeset/fuzzy-crews-boil.md

```diff
@@ -0,0 +1,6 @@
+---
+"@redocly/openapi-core": patch
+"@redocly/cli": patch
+---
+
+Resolved an issue where overrides for the severity of configurable scorecard rules were ignored.
```

Comment:
Looks like it's not just about scorecard rules but configurable rules in general.